.travis.yml

```diff
@@ -4,31 +4,47 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PERL=5.20.3.1
+    - CONDA_PERL=5.22.0.1
+    - CONDA_PERL=5.22.2.1
```

Comment:
Sneaky Perl is getting added several times. Can we just pin Perl to `5.20.3.1` in `build` as this doesn't depend on Perl? That should get rid of this unneeded build matrix.

xref: https://github.com/conda-forge/conda-smithy/issues/415